components/tidb_query_datatype/src/codec/batch/lazy_column.rs

```diff
@@ -208,6 +208,8 @@ impl LazyBatchColumn {
             }
         }
 
+        ctx.n_bytes += raw_vec_len;
```

Comment:
is the n_bytes supposed to count the memory of raw_vec or the vec in decoded_column?
In anycase, why the memory usage is just same as `raw_vec_len`

Reply:
ctx.n_bytes  in this function counts only the row bytes.

Reply:
Fixed.